### ISSUE TITLE
Resizable Rows

### ISSUE THREAD

**Issue by andreivanea** [olifolkerd/tabulator#188]
In the current version of the Documentation, I couldn't find a reference for allowing rows to be resizable. There are cases when a cell can contain a lot of text. Rendering a really tall row could harm usability. My issue is two-fold:

1. is there a way of specifying the maximal (initial) height of rows, and
2. is there a way of making the rows resizable?

And now some details. Regarding _1_, consider a grid having a column which stores comments/annotations for the specific row (but any larger text in general -- 2-3 paragraphs). In such cases it would help to be able to set a max height on the rows, so that the data does not mess up the design, at least on the first load.

Regarding _2_, the user could resize the row to show/hide the content and have a better view of a specific row (when increasing the height) or of all rows (when decreasing).

**Comment by olifolkerd** [comment]
Hey,

You can use the ***textarea*** formatter to generate a textbox that resizes to fit multi line text:

```js
$("#example-table").tabulator({
    columns:[
        {title:"Comments", field:"comments", formatter:"textarea"},
    ],
});
```


At the moment there is no way to resize rows through Tabulator, but i like the idea. i will put together a demo for you of how to achieve this yourself and will look at adding it to tabulator in the next release

Cheers

Oli

**Comment by olifolkerd** [comment]
Hey,

Just a quick note to let you know that this feature will be available in the upcoming 3.0 release later in May!

Cheers

Oli

**Comment by olifolkerd** [comment]
Hey @andreivanea 

You will be happy to hear i have pushed an update to the 3.4 branch with this change in.

You will be able to use the ***resizableRows*** to enable this feature:

```js
$("#example-table").tabulator({
    resizableRows:true, // this option takes a boolean value (default = false)
});
```

Version 3.4 should be released in the next couple of weeks.

Cheers

Oli
